.gitignore

```diff
@@ -25,6 +25,11 @@ hs_err_pid*
 
 
 
+## IntelliJ build
+out/
```

Comment:
This should go into your own global `.gitignore`.

Reply:
Alright